# Solved Python3 issues

## UnicodeEncodeError: 'ascii' codec can't encode character '\xe9' in position 10: ordinal not in range(128)
`apt install language-pack-fr-base`  
`export LC_ALL="fr_FR.UTF-8"`
